k8s/filings-ui.dc-srvc-rt-cmap-kcmap.param

```diff
@@ -21,14 +21,14 @@ MOUNT_PATH="/var/www/html/business/config"
 DOMAIN="dev.bcregistry.ca"
 PATH="/business"
 # For config maps
-BUSINESSES_URL="https://dev.bcregistry.ca/business/auth/"
+BUSINESSES_URL="https://dev.bcregistry.ca/business"
```

Comment:
This URL should point to the Businesses home page.
Your change will load the Filings UI, which will then load Auth Web (2 steps).
Please revert.

(Yes, we should possibly rename this and/or clean up our usage of this URL, but that's outside the scope of this ticket.)

Reply:
ok.